src/Framework/ref/Microsoft.AspNetCore.App.Ref.Internal.csproj

```diff
@@ -8,7 +7,11 @@
     <IsPackable Condition="'$(IsTargetingPackBuilding)' == 'false'">false</IsPackable>
     <PackageId>$(TargetingPackName).Internal</PackageId>
     <VersionPrefix>$(TargetingPackVersionPrefix)</VersionPrefix>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IncludeSymbols>false</IncludeSymbols>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
```

Comment:
Side question: Should this and the main `Microsoft.AspNetCore.App.Ref` project also set `<IsReferenceAssemblyProject>false</IsReferenceAssemblyProject>`❔ No urgency here and I'm not asking for a change in this PR but most of the special cases for `$(IsReferenceAssemblyProject)` projects are not necessary here (all but referencing reference assemblies). Probably better to put these in the `$(IsImplementationProject)` category (unless of course some of _those_ special cases break things 😈).

Reply:
As far as I can tell, it shouldn't change how these projects build either way. Mostly it affects AssemblyVersions and compilation items, which don't apply here. There's a condition in versions.props that ensures that we use the `.0` versions of CoreFx dependencies in ref projects, but it also checks for if the project is named `Microsoft.AspNetCore.App.Ref`, so either setting should be fine.